examples/python/aws-eks/Pipfile

```diff
@@ -0,0 +1,11 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[requires]
+python_version = "3.7"
+
+[packages]
+constructs="*"
+cdktf="*"
```

Comment:
Please use the relative reference here `cdktf = {path = "./../../../dist/python/cdktf-0.0.0-py3-none-any.whl"}`.